packages/dd-trace/src/config.js

```diff
@@ -163,6 +164,36 @@ class Config {
       process.env.DD_TRACE_EXPERIMENTAL_TRACEPARENT_ENABLED,
       false
     )
+    const DD_TRACE_PROPAGATION_STYLE_INJECT = coalesce(
+      options.tracePropagationStyle && options.tracePropagationStyle.inject,
+      process.env.DD_TRACE_PROPAGATION_STYLE_INJECT,
+      // TODO: change this to traceparent,datadog
+      // This can't be done as a semver-minor until tracestate support also exists
+      'datadog'
+    ).split(',').reduce((m, v) => {
```

Comment:
Let's trim too.

Reply:
Actually, I just noticed that this is applied to both the environment variable and the programmatic option, but the programmation option would be an actual array.

Reply:
It's not intended to be an array, it's a string. Perhaps we could support both here though?

Reply:
Made it support both forms.

Reply:
For the environment variable it makes sense that it's a string, but programatically you'd never really want to use the string form.